pom.xml

```diff
@@ -883,6 +883,11 @@
                 <artifactId>mockito-junit-jupiter</artifactId>
                 <version>${mockito.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apiguardian</groupId>
+                <artifactId>apiguardian-api</artifactId>
+                <version>1.1.2</version>
+            </dependency>
```

Comment:
What is this dependency used for? Is it some transitive dependency required to update JUnit?

Reply:
It is a transitive dependency. This is easy way an more stable than exclusions. It will only be import if used in a local project.